### PR TITLE
Add resource scalingo_notifier and data_source scalingo_notification_platform

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: scalingo/data_scalingo_notification_platform.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+			"logo_url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"available_event_ids": {
+				Type:     schema.TypeList,
```
Comment (EtienneM) [inline scalingo/data_scalingo_notification_platform.go]: Isn't it supposed to be a `TypeSet`?
From https://www.terraform.io/docs/extend/schemas/schema-types.html#typelist:
> Used to represent an ordered collection of items

From https://www.terraform.io/docs/extend/schemas/schema-types.html#typeset
> TypeSet implements set behavior and is used to represent an unordered collection of items

File: scalingo/data_scalingo_notification_platform.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+	name, ok := d.Get("name").(string)
+	if !ok || name == "" {
+		return errors.New("name attribute is mandatory")
+	}
+
+	var selected *scalingo.NotificationPlatform
+	for _, p := range platforms {
+		if p.Name == name {
+			selected = p
```
Comment (EtienneM) [inline scalingo/data_scalingo_notification_platform.go]:
```suggestion
			selected = p
			break
```

File: scalingo/resource_scalingo_notifier.go [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+	d.Set("app", split[0])
+	d.SetId(split[1])
+
+	return []*schema.ResourceData{d}, nil
+}
+
+func readNotifierParamsFromResource(d *schema.ResourceData, c scalingo.API) (scalingo.NotifierParams, error) {
+	types, err := c.EventTypesList()
+	if err != nil {
+		return scalingo.NotifierParams{}, fmt.Errorf("fail to list even types: %v", err)
```
Comment (EtienneM) [inline scalingo/resource_scalingo_notifier.go]:
```suggestion
		return scalingo.NotifierParams{}, fmt.Errorf("fail to list event types: %v", err)
```

File: scalingo/resource_scalingo_notifier.go [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+		// For Slack/Webhook/Rocket.Chat
+		WebhookURL: d.Get("webhook_url").(string),
+	}, nil
+}
+
+func setFromScNotifier(d *schema.ResourceData, c scalingo.API, notifier *scalingo.Notifier) error {
+	types, err := c.EventTypesList()
+	if err != nil {
+		return fmt.Errorf("fail to list even types: %v", err)
```
Comment (EtienneM) [inline scalingo/resource_scalingo_notifier.go]:
```suggestion
		return fmt.Errorf("fail to list event types: %v", err)
```

File: scalingo/data_scalingo_notification_platform.go (hunk 1 above)
Comment (Soulou) [reply scalingo/data_scalingo_notification_platform.go]: You're right, we don't ensure the order

Comment (Soulou) [reply scalingo/data_scalingo_notification_platform.go]: done

**Comment by Soulou** [comment]
Review fixes done

**Pull request merged**